app/src/main/res/values-de/strings.xml

```diff
@@ -27,7 +27,7 @@
     <string name="seeker_success_title">"Fertig!"</string>
     <string name="seeker_success_message">"Die Anfrage wurde erfolgreich aufgegeben."</string>
     <string name="seeker_error_title">"Fehler!"</string>
-    <string name="seeker_error_message">"Beim Absenden der Anfrage ist ein Problem aufgetreten. Bitte versuche erneut."</string>
+    <string name="seeker_error_message">"Beim Absenden der Anfrage ist ein Problem aufgetreten. Bitte versuchen Sie es erneut."</string>
```

Comment:
We should use the "Du" speech in german. An explicit pr for that will follow with issue #65